src/components/RubberStampOverlay/RubberStampOverlay.scss

```diff
@@ -3,43 +3,195 @@
+      background: var(--popup-button-hover);
+    }
+
+    img {
+      width: 100%;
+    }
+  }
+
+  .stamp-row {
+    display: flex;
+    flex-direction: row;
+    height: 45px;
+    justify-content: space-between;
+    padding-bottom: 4px;
+
+    @include mobile {
+      justify-content: center;
+    }
+    .icon {
       display: flex;
       align-items: center;
+      justify-content: center;
+      width: 34px;
+      height: 45px;
+      border-radius: 4px;
+      cursor: pointer;
+
+      &:hover {
+        background: var(--tools-overlay-button-hover);
+      }
+    }
+
+
+  }
+  .stamp-row-content {
+    width: 162px;
+    border-radius: 4px;
+    padding: 2px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    cursor: pointer;
+    margin-right: 8px;
+
+    img {
+      max-width: 100%;
+      max-height: 100%;
+    }
+    &:hover {
+      background: var(--popup-button-hover);
+    }
+    &.interactable {
+      &:hover {
+        background: var(--tools-overlay-button-hover);
+      }
+      &.active {
+        background: var(--tools-overlay-button-active);
+      }
+    }
+
+    &.active {
+      background: var(--tools-overlay-button-active);
+    }
+  }
+
+  .stamp-row-content.add-btn {
+    @include mobile {
+      width: 90vw;
+      margin-right: 0px;
+    }
+  }
+  .tab-header {
+    @include mobile {
+      width: 300px;
+      margin: auto;
+    }
+  }
+  .tab-list {
+    display: flex;
+    margin-bottom: 16px;
+    border-radius: 4px;
+    border: 1px solid var(--border);
+    height: 28px;
+    color: var(--text-color);
+    margin-bottom: 8px;
+    width: 100%;
+    cursor: pointer;
+
+    .tab-options-button {
+      text-align: center;
+      vertical-align: middle;
+      line-height: 24px;
+      text-overflow: ellipsis;
+      overflow: hidden;
+      white-space: nowrap;
+      flex: 1;
+      border-radius: 0px;
+
+      &:first-child {
+        border-bottom-left-radius: 4px;
+        border-top-left-radius: 4px;
+      }
+
+      &:last-child {
+        border-bottom-right-radius: 4px;
+        border-top-right-radius: 4px;
+      }
 
       &:hover {
         background: var(--popup-button-hover);
       }
 
-      img {
-        width: 100%;
+      &.selected {
+        background: var(--popup-button-active);
       }
     }
+
+    .tab-options-divider {
+      width: 1px;
+      background: var(--divider);
+    }
+  }
+
+
+  .add-custom-stamp-button {
+    border: 1px solid var(--icon-color);
+    color: var(--text-color);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: 32px;
+    width: 196px;
+    border-radius: 4px 4px 4px 4px;
+    font-size: 16px;
+    cursor: pointer;
+    margin-bottom: 10px;
+
+    &.disabled {
+      border: 1px solid #E1E1E3;
+      color: #E1E1E3;
+      cursor: default;
+    }
+
+    &.enabled:hover {
+      background: #17324C;
+      border: 1px solid #17324C;
+      color: #FFFFFF;
+    }
   }
```

Comment:
This css class (.add-custom-stamp-button) looks dead nothing is using this. Remove it?